include/wx/generic/private/markuptext.h

```diff
@@ -84,10 +84,14 @@ class WXDLLIMPEXP_CORE wxMarkupText
     // The meaning of the flags here is different than in the overload above:
     // they're passed to DrawItemText() and Render_ShowAccels is not supported
     // here.
+    //
+    // Currently the only supported ellipsize modes are wxELLIPSIZE_NONE and
+    // wxELLIPSIZE_END, the others are treated as wxELLIPSIZE_NONE.
```

Comment:
It's probably not really important in practice as the other modes should be used vanishingly rarely (although I think I did use `MIDDLE` with wxDVC once for a column containing file names, in order to make sure the extension is visible), but I think it'd be better to map `BEGIN` and `MIDDLE` to `END` rather than `NONE`, wouldn't it?